lib/actions/lookupObject.js

```diff
@@ -22,9 +21,9 @@ module.exports.getMetaModel = async function getMetaModel(configuration) {
  *
  * @param configuration
  */
-module.exports.getLookupFieldsModel = async function getLookupFieldsModel(configuration) {
-  const metaLoader = new MetaLoader(configuration);
-  return await metaLoader.getLookupFieldsModelWithTypeOfSearch(configuration.typeOfSearch);
+module.exports.getLookupFieldsModel = function getLookupFieldsModel(configuration) {
+  const metaLoader = new MetaLoader(configuration, this);
+  return metaLoader.getLookupFieldsModelWithTypeOfSearch(configuration.typeOfSearch);
```

Comment:
Remove `await`, why?

Reply:
The same explanation. Actually here is might be a little better explanation.
https://eslint.org/docs/rules/no-return-await